float32 center_x
float32 center_y
float32 center_z
float32 occupancy
